babyProject/babyProject/Views/ChannelViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  ChannelViewController.swift
+//  babyProject
+//
+//  Created by Otto Linden on 24.4.2021.
+//
+
+import UIKit
+import MessageKit
+import InputBarAccessoryView
+import Combine
+import StompClientLib
+
+class ChannelViewController: MessagesViewController, MessagesDataSource, MessagesLayoutDelegate, MessagesDisplayDelegate {
+
+    private var cancellable: AnyCancellable?
+
+    let currentUser = Sender(senderId: "1", displayName: "iOS-User-\(UUID().uuidString)")
+    var _messages = [Message]()
+    var messages: [Message] {
+        get {
+            return _messages
+        } set {
+            _messages = newValue
+            messagesCollectionView.reloadData()
+        }
+    }
+    var channel: bpChannel?
+
+    var socketClient = StompClientLib()
+    lazy var url = NSURL(string: "https://api.turkuforge.fi/connect/websocket")!
```

Comment:
Might be a good idea to add a comment here
```suggestion
    // Currently hardcoded url do to issues with API outputting two identical JSON fields `_links`
    lazy var url = NSURL(string: "https://api.turkuforge.fi/connect/websocket")!
```